core/src/views.py

```
@@ -0,0 +1,355 @@
+        is_published = self.request.query_params.get('is_published')
+        if is_published is not None:
+            queryset = queryset.filter(is_published=is_published.lower() == 'true')
+
+        return queryset
+
+    def perform_create(self, serializer):
+        """Set created_by to current user"""
+        serializer.save(created_by=self.request.user)
+
+    @action(detail=True, methods=['post'])
+    def validate(self, request, pk=None):
```

Ruff (0.14.5)

Unused method argument: `pk`

(ARG002)